simulator.cpp

```diff
@@ -202,4 +200,14 @@ int AgentSpecificSimulator::get_lives() {
     return simulator_ptr_->get_lives();
 }
 
+void AgentSpecificSimulator::init_screen() {
+    activate_my_agent();
+    GameSimulator::init_screen();
+}
+
+void AgentSpecificSimulator::make_context_screens() {
+    activate_my_agent();
+    GameSimulator::make_context_screens();
+}
+
```

Comment:
These two functions don't have to be overridden, because get_screen() is overridden.